src/buildCommon.js

```diff
@@ -243,6 +243,15 @@ const canCombine = (prev: SymbolNode, next: SymbolNode) => {
         return false;
     }
 
+    // If prev and next both are just "mbin"s or "mord"s we don't combine them
+    // so that the proper spacing can be preserved.
+    if (prev.classes.length === 1) {
+        const cls = prev.classes[0];
+        if (cls === "mbin" || cls === "mord") {
+            return false;
+        }
+    }
```

Comment:
@edemaine are there any other classes that we should do this for?

Reply:
I'm going to merge this as is and if we find other classes that need to be handled this way we can add them later.